content/presence-occupancy/presence.textile

```diff
@@ -0,0 +1,702 @@
+---
+title: Presence
+meta_description: "Presence events provide clients with information about the status of other clients 'present' on a channel"
+languages:
+  - javascript
+  - nodejs
+  - php
+  - python
+  - ruby
+  - java
+  - swift
+  - objc
+  - csharp
+  - go
+---
+
+Presence enables clients to be aware of other clients that are currently "present" on a channel. Each member present on a channel has a unique self-assigned client identifier and system-assigned connection identifier, along with an optional payload that can be used to describe the member's status or attributes. Presence allows you to quickly build apps such as chat rooms and multiplayer games by automatically keeping track of who is present in real time across any device.
```

Comment:
Intro needs to mention the difference between REST and realtime interfaces. I think this is also missing a suggested caveat about requiring clients to be identified.